main/saturn-node.js

```diff
@@ -239,6 +253,17 @@ function handleActivityLogs (ctx, text) {
     })
 }
 
+async function fetchJobCount (/** @type {Context} */ ctx) {
+  const res = await request(apiUrl + 'stats', { throwOnError: true })
+  const stats = await res.body.json()
+  const jobCount = stats.NSuccessfulRetrievals
```

Comment:
```suggestion
  const jobCount = stats?.NSuccessfulRetrievals
```

Just in case, since we're also defensive below